docker/Dockerfile

```diff
@@ -95,6 +95,13 @@ RUN export OPENSSL_ROOT=openssl-1.1.1q && \
     export OPENSSL_DOWNLOAD_URL=https://www.openssl.org/source && \
     manylinux-entrypoint /build_scripts/build-openssl.sh
 
+COPY build_scripts/build-tcltk.sh /build_scripts/
+RUN export TCL_ROOT=tcl8.6.12 && export TK_ROOT=tk8.6.12 \
+    export TCL_HASH=26c995dd0f167e48b11961d891ee555f680c175f7173ff8cb829f4ebcde4c1a6 && \
+    export TK_HASH=12395c1f3fcb6bed2938689f797ea3cdf41ed5cb6c4766eec8ac949560310630 && \
```

Comment:
This needs to be updated on patch releases of TCL/TK.
Please update `update_native_dependencies.py` to handle this.

Reply:
@mayeut Apparently I cannot update from sourceforge using lastversion. I tied via command line too.
The initial work for this is at https://github.com/marcelotduarte/manylinux/commit/823e114ca421448f877d1cdb8078509f29ca58d1
Can you take a look?

In the SF project site it shows 8.6.11 as the last version:
![image](https://user-images.githubusercontent.com/12752334/187073225-07299c55-ce77-4894-9371-eef71436dc88.png)

Reply:
You can use tcl/tk GitHub mirror to check the latest version.
```
lastversion tcltk/tcl --only core-8-6-
8.6.12
```

Reply:
Wow. Thanks!
Done.